superset/assets/javascripts/logger.js

```diff
@@ -0,0 +1,79 @@
+import $ from 'jquery';
+
+export const LOG_ACTIONS_LOAD_DASHBOARD = 'load_dashboard';
+export const LOG_ACTIONS_LOAD_EXPLORE = 'load_explore';
+export const LOAD_EVENT = 'load';
```

Comment:
it seem strange to add `LOG_ACTIONS_` as a prefix for actions but nothing for event names. what do you think of also adding `LOG_EVENTS_` as a prefix for this?

Reply:
fixed.